http/vibe/http/auth/basic_auth.d

```diff
@@ -105,15 +105,4 @@ bool checkBasicAuth(scope HTTPServerRequest req, scope PasswordVerifyCallback pw
 	return false;
 }
 
-
-/**
-	Augments the given HTTP request with an HTTP Basic Auth header.
-*/
-void addBasicAuth(scope HTTPRequest req, string user, string password)
```

Comment:
This is a public function. You can't just remove it. But you can introduce a `deprecated alias`

Reply:
Fixed